home.html

```diff
@@ -1,207 +1,255 @@
+        </div>
+      </div>
+
+      <!-- Content section with instructions text -->
+      <div class="content-section">
+        <div class="toggle-item">
+          <h2 class="page-title">About</h2>
+        </div>
+        <div class="text-box toggle-item">
+          <p class="text-content">
+            'MasterMind Gems' is a recreation of the classic board game,
+            MasterMind.<br /><br />We created this game keeping in mind the
+            nostalgia most of us have for traditional board games. It may not be
+            as fun as beating a friend at the physical game, however, we do hope
+            The CodeMaster gets you as close to the real game.<br /><br />
+            This is a hobby mini-project and we welcome all feedback, even if
+            you just want to tell us how disappointing we are!!<br /><br />Do
+            let us know what you think of the game or just say Hi on any of the
+            links below!
+          </p>
+        </div>
+        <a class="nav-link" href="#main-pg" target="">
+          <div
+            class="menu-title load-item"
+            id="back_3"
+            onclick="hideAndShow('about-pg','main-pg')"
+          >
+            Back
+          </div>
+        </a>
+      </div>
+
+      <!-- Footer section -->
+      <footer class="footer-content">
+        <div class="information">
+          <!-- <hr class="line-divider"> -->
+          <div class="other-links"></div>
+          <div class="social-media toggle-item">
+            <a
+              class="social-link"
+              href="https://twitter.com/luizoutlets"
+              title="twitter"
+            >
+              <img
+                class="social-icon"
+                src="images/social_icons/twitter-light-wheat-40px.png"
+                alt="Twitter link"
+              />
+            </a>
+            <a class="social-link" href="mailto:luletco@gmail.com" title="mail">
+              <img
+                class="social-icon"
+                src="images/social_icons/mail-light-wheat-40px.png"
+                alt="email link"
+              />
+            </a>
+            <a
+              class="social-link"
+              href="https://github.com/lulets"
+              title="github"
+            >
+              <img
+                class="social-icon"
+                src="images/social_icons/github-light-wheat-40px.png"
+                alt="Github link"
+              />
+            </a>
+          </div>
+
+          <span class="person-info">&copy;2019 &nbsp;Loet</span>
+        </div>
+      </footer>
+    </div>
+
+    <!-- PRELOADER -->
+    <div id="preloader" class="load-wrapper">
+      <div class="loader-back"></div>
+
+      <div id="loader-box" class="box-position">
+        <div class="title-header">
+          <img class="game-title" src="images/07_Title.png" />
+        </div>
+      </div>
+    </div>
+  </body>
+  <script type="module" src="js/main_pg_preload.js"></script>

```

Comment:
If we would like to go with the idea of having a `utils` class, we will need to change the type to be `module` at this point.